src/main/java/com/clanki/ui/Ui.java

```diff
@@ -46,4 +46,16 @@ public void printByeMessage() {
     public void printInvalidInput() {
         System.out.println("The input is in an incorrect format.");
     }
+
+    /**
+     * reused from Kong Dehao ip
+     * for general printing to console
+     *
+     * @param message multiple strings to be shown to user
+     */
+    public void print(String... message) {
```

Comment:
I think a name like `printlnSeveralStrings` would be a better name. `print` can be mistook as printing something on the same line, and I think the `text-ui-test` bug above is due to this exact mistake.